ramips: fix IMAGE_SIZE of HC5x6: fix image size of HC5x61

DragonBluep opened the pull request:
Fixes: df1e5d6 ("ramips: fix partition layout of hiwifi hc5x61")

"firmware" partition size defined in the device tree file is 0xf70000,
so the right IMAGE_SIZE is 15808k

adschm commented:
Consider replacing "image size" with "IMAGE_SIZE" in the commit message, so it's obvious this is about that specific variable.

DragonBluep committed:
ramips: fix IMAGE_SIZE of HC5x6: fix image size of HC5x61

Fixes: df1e5d646345 ("ramips: fix partition layout of hiwifi hc5x61")

"firmware" partition size defined in the device tree file is 0xf70000,
so the right IMAGE_SIZE is 15808k

Signed-off-by: Shiji Yang <yangshiji66@qq.com>

hauke commented:
Please use your real name and the same email address as commit author and in the signed-off-by line.
Does this fix a problem added in df1e5d646345? If so please add this fixes line to the commit:
```
Fixes: df1e5d646345 ("ramips: fix partition layout of hiwifi hc5x61")
```

DragonBluep commented:
> Please use your real name and the same email address as commit author and in the signed-off-by line.
> Does this fix a problem added in [df1e5d6](https://github.com/openwrt/openwrt/commit/df1e5d6463459637e2c1b39b11a05a45ed2322d4)? If so please add this fixes line to the commit:
> 
> ```
> Fixes: df1e5d646345 ("ramips: fix partition layout of hiwifi hc5x61")
> ```

OK, I have corrected it. @hauke

hauke commented:
Thank you for the pull request, I applied it to master.
